File: Assets/AltUnityTester/Examples/Test/Editor/Driver/TestsHelper.cs

namespace Altom.AltUnityDriver.Tests
{
    public class TestsHelper
    {
        public static int GetAltUnityDriverPort()
        {
            string port = System.Environment.GetEnvironmentVariable("ALTUNITYDRIVER_PORT");
            if (!string.IsNullOrEmpty(port))
            {
                return int.Parse(port);
            }

            return 13000;
        }

        public static string GetAltUnityDriverHost()
        {
            string host = System.Environment.GetEnvironmentVariable("ALTUNITYDRIVER_HOST");
            if (!string.IsNullOrEmpty(host))
            {
                return host;
            }

            return "127.0.0.1";
        }
    }
}

File: Bindings~/java/src/test/java/ro/altom/altunitytester/TestsHelper.java
package ro.altom.altunitytester;

public class TestsHelper {
    public static int GetAltUnityDriverPort() {
        String port = System.getenv("ALTUNITYDRIVER_PORT");
        if (port != null && !port.equals("")) {
            return Integer.parseInt(port);
        }

        return 13000;
    }

    public static String GetAltUnityDriverHost() {
        String host = System.getenv("ALTUNITYDRIVER_HOST");
        if (host != null && !host.equals("")) {
            return host;
        }

        return "127.0.0.1";
    }
}
